BridgeSDKTests/_SBBBridgeObject_test.h

```diff
@@ -33,12 +33,15 @@
 #import <Foundation/Foundation.h>
 #import "ModelObject.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @protocol _SBBBridgeObject_test
 
 @end
 
 @interface _SBBBridgeObject_test : ModelObject
 
-@property (nonatomic, strong, readonly) NSString* type;
+@property (nullable, nonatomic, strong, readonly) NSString* type;
```

Comment:
non-null?

Reply:
it's a test object and objective-c seems to pretty much ignore the nullability specifiers, but sure